openpathsampling/volume.py

```diff
@@ -406,7 +419,16 @@ def __init__(self, collectivevariable, lambda_min = 0.0, lambda_max = 1.0,
 
     def do_wrap(self, value):
         """Wraps `value` into the periodic domain."""
-        return ((value-self._period_shift) % self._period_len) + self._period_shift
+        val = value - self._period_shift
+        if val > val * 0:
+            return value - int(val / self._period_len) * self._period_len
+        else:
+            wrapped = value + int((self._period_len - val) / self._period_len) \
+                * self._period_len
+            if wrapped >= self._period_len:
+                wrapped -= self._period_len
+
+            return wrapped
```

Comment:
can you add a comment here to remind future-us why you did this? I get why, from your comments in the PR, but I'm pretty sure I'd look at this later and go "WTF?"

(As easy as "equivalent to: $OLD_STUFF; needed because unit don't like %")